### PR TITLE
Added to views

### PR DESCRIPTION
**Pull request by st8991**
Addaccount and accountManagement added to views,
as well as small adjust to models and the templates

### CODE REVIEW

**Review by turkogl3 — APPROVED** [review]
looks good

**Review by JPeterson96 — APPROVED** [review]

File: project_app/views.py [hunk 1]
```diff
@@ -34,4 +33,26 @@ class Home(View):
...
+        emails = list(map(UserModel.email))
+        return render(request, "accountManagement.html", {"emails": emails})
+
+
+class AddAccount(View):
+    def get(self, request):
+        return render(request, "addAccountPage.html")
+
+    def post(self, request):
+        driver = Driver()
```
Comment (miladmasoodi) [inline project_app/views.py]: driver = request.session["driver"] # to fetch the driver rather than init a new one

**Review by miladmasoodi — APPROVED** [review]
looks good

**Review by vitky25 — COMMENTED** [review]
Nicely done. I'll have to rephrase the acceptance tests' url response.

**Pull request merged**